Projects/App/Sources/UI/User/Login/Views/ThirdPartyLoginViewController.swift

```diff
@@ -0,0 +1,110 @@
+//
+//  ProfilePlaceHolder.swift
+//  App
+//
+//  Created by Lee Myeonghwan on 2022/10/07.
+//  Copyright © 2022 zesty. All rights reserved.
+//
+
+import AuthenticationServices
+import UIKit
+import SnapKit
+import DesignSystem
+
+final class ThirdPartyLoginViewController: UIViewController {
+    
+    // MARK: Properties
+    
+    private let titleStackView = UIStackView()
+    private let titleLabel = UILabel()
+    private let subtitleLabel = UILabel()
+    private let characterImageView = UIImageView()
+    private let loginStackView = UIStackView()
+    private let kakaoLoginButton = UIButton()
+    private let appleLoginButton = UIButton()
+    
+    // MARK: LifeCycle
+    
+    override func viewDidLoad() {
+        configureUI()
+        configureLayout()
+    }
+    
+    // MARK: Function
+    
+    @objc func kakaoLoginButtonClicked() {
+        navigationController?.pushViewController(NickNameInputViewController(), animated: true)
+    }
+    
+    @objc func appleLoginButtonClicked() {
+        navigationController?.pushViewController(NickNameInputViewController(), animated: true)
+    }
+    
+}
+
+extension ThirdPartyLoginViewController {
+    
+    // MARK: UI Function
+    
+    private func configureUI() {
+        view.backgroundColor = .white
+        
+        navigationItem.backBarButtonItem = UIBarButtonItem(title: "", style: .plain, target: nil, action: nil)
+        navigationItem.backBarButtonItem?.tintColor = .black
+        
+        titleStackView.axis = .vertical
+        titleStackView.spacing = 12
+        
+        titleLabel.text = "반가워요, ZESTER"
+        titleLabel.font = .systemFont(ofSize: 26)
+        
+        subtitleLabel.text = "로그인하여 모든 맛집을 확인하세요."
+        subtitleLabel.textColor = .gray
+        subtitleLabel.font = .preferredFont(forTextStyle: .callout)
+        
+        characterImageView.image = UIImage(.img_zesterthree)
+        
+        loginStackView.axis = .vertical
+        loginStackView.spacing = 20
+        
+        kakaoLoginButton.addTarget(self, action: #selector(kakaoLoginButtonClicked), for: .touchUpInside)
+        kakaoLoginButton.setImage(UIImage(.btn_kakaologin), for: .normal)
+        kakaoLoginButton.imageView?.contentMode = .scaleAspectFit
+        
+        appleLoginButton.addTarget(self, action: #selector(appleLoginButtonClicked), for: .touchUpInside)
+        appleLoginButton.setImage(UIImage(.btn_applelogin), for: .normal)
+        appleLoginButton.imageView?.contentMode = .scaleAspectFit
+    }
+    
+    private func configureLayout() {
+        view.addSubviews([titleStackView, characterImageView, loginStackView])
```

Comment:
[P2]

```suggestion
    
    private func createLayout() {
        view.addSubviews([titleStackView, characterImageView, loginStackView])
```

통일한 UI 메서드 명 규칙 준수 부탁드려요!

Reply:
이걸 빠트렸네요😿 수정완료! [30d3a3a](https://github.com/DeveloperAcademy-POSTECH/MacC-TEAM-ZESTY/pull/22/commits/30d3a3afde086f0b5d6df6a95bc69c1b6c15329f)